projects/js-packages/shared-extension-utils/package.json

```diff
@@ -17,6 +17,7 @@
 	},
 	"devDependencies": {
 		"jetpack-js-test-runner": "workspace:*",
+		"lodash": "4.17.21",
```

Comment:
We can get rid of this now.

Reply:
We need this for other files like [get-jetpack-extension-availability](https://github.com/Automattic/jetpack/blob/b94c0bd9cafa1f05b4f837433765e01b87a2833d/projects/js-packages/shared-extension-utils/src/get-jetpack-extension-availability.js#L26). I will have to add it back again even if I remove it from this PR.

Reply:
If it's just for `get` then we can remove the dependency there too. That instance could become something like:

```
const details = data?.available_blocks?.[name]?.details ?? [];
```

Reply:
That sounds about right. Let me remove it for now. We can add it back if required.